src/Compilers/CSharp/Test/Syntax/LexicalAndXml/PreprocessorTests.cs

```diff
@@ -244,6 +248,12 @@ private void VerifyDirectivesSpecial(CSharpSyntaxNode node, params DirectiveInfo
                             Assert.Equal(exp.Text, ld.File.Value);
                         }
                         break;
+                    case SyntaxKind.NonNullDirectiveTrivia:
+                        if (null != exp.Text)
```

Comment:
>if (null != exp.Text) [](start = 24, length = 21)

Can we assert something otherwise? For example, that DisableOrRestoreKeyword is missing?